docs/exts/exampleinclude.py

```diff
@@ -252,5 +252,5 @@ def setup(app):
     app.add_config_value("exampleinclude_sourceroot", None, "env")
     if not airflow_theme_is_available:
         # Sphinx airflow theme has its own styles.
-        app.add_stylesheet('exampleinclude.css')
+        app.add_css_file('exampleinclude.css')
```

Comment:
BTW this fixes a deprecation warning.

Reply:
I added the info to the commit msg.

Reply:
Can you do it in a separate PR? We would like to have these changes in Airflow 1.10 too.

Reply:
yeah would be good to have it in a separate PR

Reply:
Good point - will do it separately.